src/compiler-host.js

```diff
@@ -239,7 +239,11 @@ export default class CompilerHost {
     if (!compiler) {
       compiler = this.fallbackCompiler;
 
-      let { code, binaryData, mimeType } = await compiler.get(filePath);
+      if (!compiler) {
+        compiler = this.getPassthroughCompiler();
```

Comment:
@paulcbetts I used this rep for repro

https://github.com/manikantag/electron-compile-css-issue